src/account/accountActions.js

```diff
@@ -39,3 +40,8 @@ export const loginSuccess = (realm: string, email: string, apiKey: string) => ({
 export const logout = () => ({
   type: LOGOUT,
 });
+
+export const saveGCM = (gcmToken) => ({
```

Comment:
Where do you intend to call this from?
The name should match more closely the action. So saveTokenGCM?